examples/webots/vacuum/vacuum_lib.scenic

```diff
@@ -0,0 +1,91 @@
+"""
+Library for generating robot vacuum scenarios.
+"""
+
+model scenic.simulators.webots.model
+
+import numpy as np
+import trimesh
+import random
+from pathlib import Path
+
+param numToys = 0
+param duration = 10
+
+## Class Definitions ##
+
+class Vacuum(WebotsObject):
+    webotsName: "IROBOT_CREATE"
+    shape: CylinderShape()
+    width: 0.335
+    length: 0.335
+    height: 0.07
+    customData: str(random.getrandbits(32)) # Random seed for robot controller
```

Comment:
It just occurred to me that while this does happen to work because it's inside a default value expression and so gets newly evaluated for each object, we shouldn't do this in example code: we tell people not to use `random` in the documentation for good reasons (e.g. simulation replays won't be deterministic). I would use something like `DiscreteRange(0, 2**32 - 1)`.

Reply:
Ah good point, I'm breaking basic Scenic rules here :facepalm: